openedx_webhooks/tasks/github.py

```diff
@@ -20,14 +20,17 @@
 from openedx_webhooks.jira_views import get_jira_custom_fields
 
 
+COVERLETTER_MARKER = ":grey_exclamation: This message was added automatically by the [OpenEdX Bot](http://github.com/edx/openedx-webhooks)"
```

Comment:
We shouldn't use this for anything but cover letters, or the `has_internal_cover_letter` function could get confused.

BTW: Markdown supports HTML, so we can use HTML comment syntax to include text in comments that won't distract human readers.

There's a comment here: <!-- Secret message! -->